test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -471,7 +471,23 @@ json_spirit::mObject fillBCTest(json_spirit::mObject const& _input, bool _allowI
     }
 
     output["blocks"] = blArray;
-    output["postState"] = fillJsonWithState(testChain.topBlock().state());
+
+    size_t totalCodeSize = 0;
+    bool isSmallState = testChain.topBlock().state().addresses().size() < 20;
+    if (isSmallState)
+    {
+        for (auto const& accAddr : testChain.topBlock().state().addresses())
+        {
+            totalCodeSize += testChain.topBlock().state().codeSize(accAddr.first);
```

Comment:
Why only code? Storage could be huge, too (I don't know if it's worth to consider in tests)

Reply:
hm. you right. although it is likely that a huge code is deployed rather then storage. but that one needs to be checked too.